OWML.ModHelper.Menus/ModPopupInput.cs

```diff
@@ -34,9 +42,7 @@ private void SetupCommands()
         {
             var listenerObject = new GameObject();
             CommandListener = listenerObject.AddComponent<ModCommandListener>();
-            CommandListener.AddToListener(InputLibrary.select);
-            CommandListener.AddToListener(InputLibrary.enter);
-            CommandListener.AddToListener(InputLibrary.enter2);
+            _commands.ForEach(command => CommandListener.AddToListener(command));
```

Comment:
Neither VS nor ReSharper suggested it, but this actually works: `_commands.ForEach(CommandListener.AddToListener);`

Reply:
done